cpp/benchmarks/io/parquet/experimental/parquet_dictionary_page_filter.cpp

```diff
@@ -94,7 +94,6 @@ std::vector<rmm::device_buffer> fetch_byte_ranges(
       return buffer;
     });
 
-  stream.synchronize_no_throw();
```

Comment:
The column chunk buffers are accessed via in stream ordered kernels internally so no need for this sync